src/node/markdownToVue.ts

```diff
@@ -56,7 +56,10 @@ export function createMarkdownToVueRenderFn(
             ? url.slice(1)
             : path.relative(root, path.resolve(dir, url))
         )
-        if (!pages.includes(resolved)) {
+        if (
+          !pages.includes(resolved) &&
+          !fs.existsSync(path.resolve(dir, 'public', `${resolved}.html`))
```

Comment:
Better to use `config.publicDir`

Reply:
Yeah, thanks. But i can not find how to use vite config in vitepress.
Can you give me a hint?

Reply:
How can i get `resolvedViteConfig` in `markdownToVue.ts`?
It seems that there is not a global configuration exported.
Should there be a better solution to get some global configuration in the function?

https://github.com/vuejs/vitepress/blob/77f144a0c75d6a238abd4a486d20c1e9a6888ab5/src/node/build/bundle.ts#L83

https://github.com/vuejs/vitepress/blob/77f144a0c75d6a238abd4a486d20c1e9a6888ab5/src/node/markdownToVue.ts#L20

Reply:
You can get it using https://vitejs.dev/guide/api-plugin.html#configresolved in plugin.ts and pass it over

Reply:
Thanks. I submitted a new commit that uses `config.publicDir` instead of the `'public'` string.